.terraform.lock.hcl

```diff
@@ -2,31 +2,32 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/cloudflare/cloudflare" {
-  version = "3.6.0"
+  version = "4.2.0"
```

Comment:
Could you update this to install version 3.33.0 of the cloudflare provider instead of 4.2.0?  [CI is failing because 4.2.0 is not compatible with terraform 0.14.7](https://github.com/equinix/terraform-metal-openshift-on-baremetal/actions/runs/4657321097/jobs/8246092702#step:5:70), and I don't think we're ready to drop support for terraform 0.14 in the scope of this PR.

Reply:
@rawkode what problem are you trying to solve with the version change for Cloudflare? Is this about the problem we chatted about regarding the requirement for API tokens in the provider definition even when that provider may not be used?

Reply:
For example: https://github.com/equinix/terraform-metal-openshift-on-baremetal/pull/18/files#diff-dfd5848fa77a04d0343891fe859286cc210473d10f0e62c7f99f0d5ecf1a9927R5

Reply:
CI has been updated to use terraform v1 and that change was merged into this PR, so I no longer have any concerns about the provider upgrade.